### PR TITLE
fix(lyrics-plus): improve detection of zh and ja

### PR DESCRIPTION
**Pull request by Deskehs**
A new pr continuing from #2184

### CODE REVIEW

**Review by kyrie25 — APPROVED** [review]

**Review by rxri — CHANGES_REQUESTED** [review]
prettify all files

File: jsHelper/expFeatures.js [hunk 1]
```diff
@@ -1,59 +1,59 @@
 (function () {
-    let overrideList,
```
Comment (kyrie25) [inline jsHelper/expFeatures.js:2]: Prettify and commit **only** the relevant files.

**Review by kyrie25 — CHANGES_REQUESTED** [review]

**Comment by Deskehs** [comment]
reverted the file to original status as requested

**Review by kyrie25 — APPROVED** [review]

File: CustomApps/lyrics-plus/Utils.js [hunk 2]
```diff
@@ -56,10 +56,26 @@ class Utils {
...
+
+		for (const character of charMatch) {
+			if (character.match(cjkRegex)) {
+				cjkCount++;
+			} else {
+				kanaCount++;
+			}
+		}
+
+		return ((kanaCount / charMatch.length - cjkCount / charMatch.length + 1) / 2) * 100 >= CONFIG.visual["ja-detect-threshold"];
```
Comment (rxri) [inline CustomApps/lyrics-plus/Utils.js:78]: can't we split this line to variables so it's easier to read?

Comment (Deskehs) [reply CustomApps/lyrics-plus/Utils.js]: just separated the percentages into it's own variables

**Comment by rxri** [comment]
Don't force push every-time you change something. It makes it harder to us to see what you're actually changing

**Comment by Deskehs** [comment]
Oh oh alright then, is there anything you would like to change?

**Review by rxri — APPROVED** [review]

**Review by rxri — APPROVED** [review]

**Pull request merged**